search/lib/search/shortest_path.ex

```diff
@@ -0,0 +1,135 @@
+defmodule Search.ShortestPath do
```

Comment:
I think I'd prefer to see the implementation code split into a separate module here.

# -4